src/sonic-ctrmgrd/ctrmgr/container

```diff
@@ -301,19 +306,25 @@ def container_kill(feature, **kwargs):
 
     init()
 
-    set_owner, _ = read_config(feature)
+    ret = SUCCESS
+    set_owner, _ , state = read_config(feature)
     current_owner, remote_state, _ = read_state(feature)
     docker_id = container_id(feature)
     remove_label = (set_owner != "local") or (current_owner != "local")
 
-    debug_msg("{}: set_owner:{} current_owner:{} remote_state:{} docker_id:{}".format(
-        feature, set_owner, current_owner, remote_state, docker_id))
+    debug_msg("{}: set_owner:{} current_owner:{} remote_state:{} docker_id:{} state:{}".format(
+        feature, set_owner, current_owner, remote_state, docker_id, state))
 
     if remove_label:
         set_label(feature, False)
 
+    if set_owner == "local":
+        if state not in ["enabled", "always_enabled"]:
+            debug_msg("{} is not enabled".format(feature))
+            return FAILURE
```

Comment:
@dgsudharsan should we return failure in this case? if just means nothing to kill and the docker command will not executed

Reply:
The failure return code is to signify the fact that the user input is wrong (docker is not running) and cannot be killed. This error code is returned when the container command is executed which is inline with many other linux commands.

Reply:
This PR is to avoid the failure.
It was "_No such container: nat_"
Would the caller (warm/fast reboot) would still fail, because of this return code?

Reply:
No. The error code will not fail the warm-reboot and its validated.
https://github.com/Azure/sonic-utilities/blob/b6ca76b4821453171d9607383b808385968eeeeb/scripts/fast-reboot#L699